src/util/cardCatalog.ts

```diff
@@ -173,8 +174,9 @@ async function loadJSONFile(filename: string, attribute: keyof Catalog) {
       readStream.pipe(parseStream);
 
       readStream.on('end', () => {
+        const fileDuration = ((Date.now() - fileStart) / 1000).toFixed(2);
         // eslint-disable-next-line no-console
-        console.info(`Loaded ${filename} into ${attribute}`);
+        console.info(`Loaded ${filename} into ${attribute} in ${fileDuration}s`);
```

Comment:
Added duration of loading files in the app start. I did not find any different in loading speeds with different versions (apart from loading via import, but that got very messy and so wasn't viable)